packages/components/COMPONENT-CHECKLIST.md

```diff
@@ -39,7 +32,7 @@ The engineering checklist has six parts: creating the feature branch, component
   - assertion text should match the content style of the other components.
   - goal is a terse invocation
 - [ ] **component style**
-  - create `component/COMPONENT_NAME.scss` in `app/styles` 
+  - create `component/COMPONENT_NAME.scss` in `app/styles`
   - add `@use` to `app/styles/@hashicorp/design-system-components.scss` (see existing code for precise syntax)
   - use design tokens wherever possible, comment where they are not
   - sizes should be in relative units
```

Comment:
```suggestion
  - sizes should be in relative units
  - use BEM for naming
```

Reply:
This is not yet defined. We have to be more subtle here.